src/styles/blocks/wrap-calendar.scss

```diff
@@ -0,0 +1,4 @@
+.wrap-calendar {
+  width: calc(#{$square-size} * 7 + #{$gap-size} * 6 + #{$calendar-pading} * 2);
```

Comment:
You can write a function for this calculations

Reply:
Yes. I can, but... as for me, it makes no sense to use a function in this case. We will not use any other variables besides these.